### DISCUSSION TITLE
How to add a new device?

### DISCUSSION THREAD

**Discussion by SFoskett** [fuatakgun/eufy_security#915] [Q&A]
I had eufy_security working fine with the original Chime and Doorbell but the new Homebase 3 is not appearing after adding it. Eufy's (bad) procedure for switching bases (from the Chime to the Homebase S380) is to remove the device in the app and pair it with the new base. This left me with a broken Doorbell (left over from the Chime) and nothing else since it can't see the new Homebase.

How do I get eufy_security to add the Homebase to Home Assistant? Or is this a eufy-security-ws problem?

I'm using the latest version of everything on a Raspberry Pi 3.

Thanks!

**Comment by pbfreak** [comment]
I have the same problem. With my old Base 2 everything works fine. But with the S380HB i can run the addon fine but no entities are showing up.

In the eufy-security-ws my cams are showing up in the Debug. But i cant get them into the Eufy_security

**Comment by fuatakgun** [comment]
Normally, if these devices are shared with your account (assuming you are using an explicit account for this) they will be created on the next restart (not sure about reload).

If you think all has been setup right, create an issue following bug template

**Comment by pbfreak** [comment]
Thx for your response! As usual the Error sits in front of the screen. I had 5 updates pending in HACS and missed to start them. (thought it runs automatically....) After Updating everything runs fine as usual
